src/nifake/metadata/functions.py

```diff
@@ -916,6 +916,7 @@
             {
                 'direction': 'in',
                 'enum': None,
+                'is_buffer': True,
                 'name': 'aString',
                 'type': 'ViChar',
```

Comment:
Wouldn't the fix be to set the type to `ViChar[]`?

Reply:
fixed